README.md

```diff
@@ -65,55 +112,93 @@ vars:
+  using_lead_history_mode_active_records: true         # false by default. Only use if you have history mode enabled.
+  using_task_history_mode_active_records: true         # false by default. Only use if you have history mode enabled.
+  using_event_history_mode_active_records: true        # false by default. Only use if you have history mode enabled.
+  using_product_2_history_mode_active_records: true    # false by default. Only use if you have history mode enabled.
+  using_order_history_mode_active_records: true        # false by default. Only use if you have history mode enabled.
+  using_opportunity_line_item_history_mode_active_records: true       # false by default. Only use if you have history mode enabled.
 ```
 
-### Disabling Models
-Your connector may not be syncing all tabes that this package references. This might be because you are excluding those tables. If you are not using those tables, you can disable the corresponding functionality in the package by specifying the variable in your dbt_project.yml. By default, all packages are assumed to be true. You only have to add variables for tables you want to disable, like so:
 
-The `salesforce__user_role_enabled` variable below refers to the `user_role` table. 
+## (Optional) Step 4: Additional Configurations
+### Change the Build Schema
+By default, this package builds the GitHub staging models within a schema titled (<target_schema> + `_stg_salesforce`) in your target database. If this is not where you would like your GitHub staging data to be written to, add the following configuration to your root `dbt_project.yml` file:
 
 ```yml
-# dbt_project.yml
-
-...
-config-version: 2
+models:
+    salesforce_source:
+      +schema: my_new_schema_name # leave blank for just the target_schema
+```
+### Adding Formula Fields as Pass Through Columns
+The source tables Fivetran syncs do not include formula fields. If your company uses them, you can generate them by referring to the [Salesforce Formula Utils](https://github.com/fivetran/dbt_salesforce_formula_utils) package. To pass through the fields, add the following configuration. We recommend confirming your formula field models successfully populate before integrating with the Salesforce package. 
 
-vars:
-  salesforce__user_role_enabled: false # Disable if you do not have the user_role table
+Include the following within your `packages.yml` file:
+```yml
+packages:
 
+  - package: fivetran/salesforce_formula_utils
+    version: [">=0.6.0", "<0.7.0"]
 ```
-The corresponding metrics from the disabled tables will not populate in downstream models.
 
-## Contributions
+Include the following within your `dbt_project.yml` file:
+```yml
+# Using the opportunity source table as example, update the opportunity variable to reference your newly created model that contains the formula fields:
+  opportunity: "{{ ref('my_opportunity_formula_table') }}"
```

Comment:
```suggestion
```yml
# Using the opportunity source table as example, update the opportunity variable to reference your newly created model that contains the formula fields:
vars:
  opportunity: "{{ ref('my_opportunity_formula_table') }}"
```